db-connector/src/main/java/uk/nhs/adaptors/connector/model/PatientAttachmentLog.java

```diff
@@ -22,10 +21,31 @@ public class PatientAttachmentLog {
     private Boolean skeleton;
     private Boolean uploaded;
     private Integer lengthNum;
-//  @NonNull patientMigrationReqId required to create
     private Integer patientMigrationReqId;
     private Integer orderNum;
     private Boolean deleted;
     private Integer postProcessedLengthNum;
     private Boolean isBase64;
+
+    public String getFileDescription() {
```

Comment:
I am just wondering if we are expecting a lot of requests of this method. If so then maybe concatenation is a better option than String format as it works faster ?

Reply:
Potentially yes. I've changed it to use concatenation.